clusterloader2/testing/density/config.yaml

```diff
@@ -25,22 +34,23 @@ steps:
 - phases:
   - namespaceRange:
       min: 1
-      max: 3
+      max: {{$namespaces}}
     replicasPerNamespace: 1
     tuningSet: Uniform100qps
     objectBundle:
     - basename: saturation-rc
       objectTemplatePath: rc.yaml
       templateFillMap:
-        Replicas: 30
+        Replicas: {{$replicas}}
         PodKind: saturationPod
 - measurements:
   - Identifier: WaitForSaturationRunning
     Method: WaitForRunningPods
     Params:
-      desiredPodCount: 90
+      desiredPodCount: {{$totalPodsRounded}}
       labelSelector: kind = saturationPod
-      timeout: 2m
+      # timeout = (nodes * 30 /20)s + 3m
```

Comment:
Update this comment in terms of #totalPods.

Reply:
Comment updated.